src/style/index.css

```diff
@@ -0,0 +1,31 @@
+@import url(./fonts.css);
+@import url(./card__img.css);
+@import url(./card__describe.css);
+@import url(./mark.css);
+@import url(./variebles.css);
+@import url(./button.css);
+
+html {
+  margin: 0;
+  padding: 0;
```

Comment:
you shouldn't reset indentation for html tag

Reply:
done